internal/internal_worker_base.go

```diff
@@ -173,17 +170,29 @@ func newBaseWorker(options baseWorkerOptions, logger *zap.Logger, metricsScope t
 			logger,
 		)
 	}
+	// for now it's default to be enabled
+	var workerUC *workerUsageCollector
```

Comment:
no need this extra line to declare variable.

```
workerUC := newWorkerUsageCollector(..)
```

Reply:
Resolved